catalog-rest-service/src/main/resources/ui/src/interface/types.d.ts

```diff
@@ -220,7 +220,7 @@ declare module 'Models' {
   export type ClientAuth = {
     authority: string;
     client_id: string;
-    provider?: 'google' | 'okta' | 'github';
+    provider?: 'google' | 'okta' | 'github' | 'auth0';
```

Comment:
any reason for github here? if we don't support it yet

Reply:
Removed Github from UI and types